Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2004-06-24
Initial Package Version: 2.6.7
Origin: Jim Gifford
Upstream Status: Sent
Description: Fixes compile issue with DB 4.2
 
--- iproute2-2.6.7/misc/arpd.c.orig	2004-06-24 18:40:32.653106672 +0000
+++ iproute2-2.6.7/misc/arpd.c	2004-06-24 18:40:51.152803395 +0000
@@ -16,7 +16,7 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <netdb.h>
-#include <db41/db_185.h>
+#include <db_185.h>
 #include <sys/ioctl.h>
 #include <sys/poll.h>
 #include <errno.h>
--- iproute2-2.6.7/misc/Makefile.orig	2004-06-24 18:40:57.779620075 +0000
+++ iproute2-2.6.7/misc/Makefile	2004-06-24 18:41:08.245751139 +0000
@@ -22,7 +22,7 @@
 	$(CC) $(CFLAGS) $(LDFLAGS) -o rtacct $(RTACCTOBJ) $(LIBNETLINK) -lm
 
 arpd: $(ARPDOBJ)
-	$(CC) $(CFLAGS) $(LDFLAGS) -o arpd $(ARPDOBJ) $(LIBNETLINK) -ldb-4.1
+	$(CC) $(CFLAGS) $(LDFLAGS) -o arpd $(ARPDOBJ) $(LIBNETLINK) -ldb
 
 rtstat: $(RTSTATOBJ)
 	$(CC) $(CFLAGS) $(LDFLAGS) -o rtstat $(RTSTATOBJ)
